src/store/index.js

```diff
@@ -13,7 +13,7 @@ import jinxesJSON from "../hatred.json";
 Vue.use(Vuex);
 
 // helper functions
-const clean = (id) => id.toLocaleLowerCase().replace(/[^a-z0-9]/g, "");
```

Comment:
This is semantically different because the ID is no longer returned in full lowercase, since the regular expression is **not** replacing alphanumeric characters. Did you intend for the semantics to change?

Reply:
I think this could cause some bugs.  If there's a capital letter it will be entirely removed.  So if someone's json has "WASHERWOMAN" the cleaned id will just be "".